be/src/storage/utils.cpp

```diff
@@ -390,4 +391,9 @@ std::string file_name(const std::string& fullpath) {
     return path.filename().string();
 }
 
+bool is_tracker_hit_hard_limit(MemTracker* tracker, int soft_limit_percent, int hard_limit_percent) {
+    int64_t limit_ratio = std::max(hard_limit_percent * 100 / soft_limit_percent, 100);
```

Comment:
shall avoid divided by zero, if the soft limit is set to 0 by the configuration.

Reply:
done